docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -307,25 +318,84 @@ Risk Factor Hierarchy
 
 .. image:: iron_risk_hierarchy.svg
 
-Cause Model Diagram
--------------------
 
-Data Description
-----------------
+Modeling Strategy
++++++++++++++++++
+
+The hemoglobin distribution for a given demographic group should be recreated 
+under the assumption that it followd the specified ensemble distribution, as 
+the python code included above instructs. For the population that is severely 
+anemic based on the hemoglobin thresholds, a certain proportion will not 
+respond to iron supplementation.
+
+The proportion of the population with mild, moderate, or severe anemia that 
+will respond to iron fortification can be measured by:
+
+.. math::
+
+	\frac{\text{prevalence}_\text{total anemia}-\text{prevalence}_\text{iron responsive anemia}}{\text{prevalence}_\text{total anemia}}
+
+Where *prevalence_iron_responsive_anemia* and *prevalence_total_anemia* are 
+equal to the severity-, age-, sex-, and location-specific prevalence summed 
+across all iron responsive anemia and all total anemia sequela IDs, 
+respectively; sequela IDs for each category are listed in the table below. 
+
+.. list-table:: Sequela IDs 
+	:widths: 5, 30, 20
+	:header-rows: 1
 
-State and Transition Data Tables
-++++++++++++++++++++++++++++++++
+	* - Anemia Severity
+	  - All Anemia Sequela
+	  - Iron Responsive Anemia Sequela
+	* - Mild
+	  - 144, 172, 177, 182, 206, 240, 438, 442, 525, 531, 537, 645, 648, 651, 654, 1004, 1008, 1012, 1016, 1020, 1024, 1028, 1032, 1057, 1061, 1065, 1069, 1079, 1089, 1099, 1106, 1120, 1361, 1373, 1385, 1397, 1409, 1421, 1433, 1445, 4952, 4955, 4976, 4985, 4988, 5009, 5018, 5027, 5036, 5051, 5063, 5075, 5087, 5099, 5111, 5123, 5225, 5228, 5249, 5252, 5273, 5276, 5393, 5567, 5579, 5606, 5627, 5648, 5651, 5654, 5678, 5699, 5702, 5705, 7202, 7214, 22989, 22990, 22991, 22992, 22993, 23030, 23034, 23038, 23042, 23046
```

Comment:
I prefer this format of your severity splits and will adopt this format for the causes I'm documenting with severity splits.